database/build.json

```diff
@@ -0,0 +1,7 @@
+{
+  "queries": [
+    "drop_customer.sql",
+    "drop_user.sql",
+    "create_user.sql"
```

Comment:
instead of create_user.sql, 
we do: 
 create_tables.sql
create_customer.sql 
 create_rider.sql 
create_manager.sql 
create_staff.sql 
.
.
.
.
create_order.sql

Reply:
I'm fine with either way, as long as not confusing.